core/common/src/main/java/alluxio/master/file/InodeSyncStreamPendingPathCollectionType.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file;
+
+/**
+ * PendingPath in InodeSyncStream type.
+ */
```

Comment:
This comment as well as the description of the property key is not quite descriptive. Can you update the comment and explain when to use STACK and when to use QUEUE?

Reply:
I think `STACK` or `QUEUE ` is too much details for the user to understand. In my opinion it's better `FIFO/LIFO` or `BFS/DFS`.